src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6783,8 +6783,14 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExpressionTreeContainsCollectionLiteral" xml:space="preserve">
     <value>An expression tree may not contain a collection literal.</value>
   </data>
-  <data name="ERR_CollectionLiteralElementNotImplemented" xml:space="preserve">
-    <value>Support for collection literal dictionary and spread elements has not been implemented.</value>
+  <data name="ERR_CollectionLiteralKeyValueOrSpreadExpected" xml:space="preserve">
+    <value>A dictionary collection literal can contain key-value pairs and spread elements only.</value>
```

Comment:
not true :)   but we can discuss.

Reply:
if there's an extension `.Add(KVP)` that was intended to work.  Or this should work for *other* dictionary types that have an `Add(KVP)`.